### PR TITLE
Header/Nav Block.

### LINKED ISSUE #2: Header/Nav
Nav

### PR DESCRIPTION
**Pull request by bstopp**
Fix #2 

Test URLs:
- After: https://feat-nav--takeda-ihs--hlxsites.hlx.page/

### CODE REVIEW

File: blocks/footer/footer.js [hunk 1]
```diff
@@ -1,25 +1,26 @@
-import { readBlockConfig, decorateIcons } from '../../scripts/lib-franklin.js';
+// import { readBlockConfig, decorateIcons } from '../../scripts/lib-franklin.js';
```
Comment (lwintch) [inline blocks/footer/footer.js:1]: Are we good to leave comments in for now?  Maybe until the next phase?

File: blocks/header/header.css [hunk 2]
```diff
@@ -1,275 +1,364 @@
-/* header and nav layout */
-header .nav-wrapper {
-  background-color: var(--background-color);
-  width: 100%;
-  z-index: 2;
-  position: fixed;
+
+header {
+  background-color: white;
```
[the remark this answers is not in the document]
Comment (bstopp) [reply blocks/header/header.css:3]: Yes, see the `styles.css` file - but in this case, i wanted literal white. I'm fine moving this to a var and referencing that.

**Review by lwintch — APPROVED** [review]
Looks solid.  There's only 1 thing in the behavior I noticed, and that's on touch devices.  The menus don't really open as the arrow controls go to the same target as the menu name when clicked.  If that's not a concern right now, everything else looks good.

File: blocks/header/header.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (lwintch) [reply blocks/header/header.css:3]: No preference here.  I thought literal white always mapped to `#fff`, but it's not a hill I'm fighting on.  No worries.

**Review by lwintch — CHANGES_REQUESTED** [review]
Removing approval over issue discussed.  Will re-approve shortly after the fix is in.

**Review by lwintch — APPROVED** [review]
Checked again, looks good.

**Pull request merged**